client/app/scripts/components/node-details/node-details-image-status.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import find from 'lodash/find';
+import map from 'lodash/map';
+import { CircularProgress } from 'weaveworks-ui-components';
+
+import { getImagesForService } from '../../actions/app-actions';
+
+const topologyWhitelist = ['services'];
```

Comment:
This view only makes sense on the 'services' k8s topology currently. Also, it relies on certain metadata being present on the node, which may not be there on other topologies.

It would be possible to also add this for individual containers, but it would involve a more tenuous integration where we somehow look up which service a container belongs to and then look up images from there.